src/css/custom.css

```diff
@@ -110,6 +110,10 @@ h2 > [class^='ti-'] {
   min-height: 52px;
 }
 
+[class*='announcementBar_'] a:hover {
+  color: var(--ifm-button-color) !important;
+}
```

Comment:
Doesn't look like this resolves as a color for me on Firefox
![image](https://user-images.githubusercontent.com/497556/178739471-5a5359db-24b2-479d-9179-50751bddfa57.png)
What's the expected result?

Reply:
Updated this, lmk what you think